apps/web-giddh/src/app/settings/integration/setting.integration.component.html

```diff
@@ -613,7 +617,7 @@ <h3 class="fs20 mrB2">ICICI Bank</h3>
                                     </sh-select>
                                 </div>
                                 <div class="custom-select select-caret pos-rel" *ngIf="item.otpType==='BANK'">
-                                    <input type="text" class="form-control" disabled placeholder="Registered" name="approvalUniqueName{{index}}" [(ngModel)]="item.approvalUniqueName">
+                                    <input type="text" class="form-control" disabled placeholder="Registered" name="approvalUniqueName{{index}}" value="">
```

Comment:
use of this field?

Reply:
actually we don't need any value in this case if OTP type BANK

Reply:
API expect us to send null if selected BANK other than need value

Reply:
Remove value attribute